scripts/third_party_size_check.py

```diff
@@ -48,17 +44,10 @@ def _get_skip_files_list():
         IOError. If failed to open app_dev.yaml in read mode.
     """
     try:
-        with python_utils.open_file('./app_dev.yaml', 'r') as app_dev_yaml:
-            try:
-                app_dev_yaml_dict = yaml.safe_load(app_dev_yaml)
-            except yaml.YAMLError as yaml_exception:
-                python_utils.PRINT(yaml_exception)
-                sys.exit(1)
-            skip_files_list = app_dev_yaml_dict.get('skip_files')
-
-            skip_files_list = [os.getcwd() + '/' + skip_files_dir
-                               for skip_files_dir in skip_files_list]
-
+        with python_utils.open_file('.gcloudignore', 'r') as gcloudignore:
+            gcloudignore_lines = gcloudignore.read().split('\n')
+            skip_files_list = [os.getcwd() + '/' + gcloudignore_line
```

Comment:
Does this properly handle comments in the file?

Also maybe use os.path.join(os.getcwd(), gcloudignore_line).

Reply:
Yep, refactored a bit.